.github/workflows/main.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         uses: ./.github/actions/ci
 
       - name: Testing Starter application development server
-        run: yarn start-server-and-test 'yarn template-starter:start' 3001 'exit 0'
+        run: yarn start-server-and-test 'yarn template-starter:start' http-get://127.0.0.1:3001 'exit 0'
```

Comment:
Just curious about where this `http-get` syntax comes from.

Reply:
It's a special notation of the "start-server-and-test" library https://github.com/bahmutov/start-server-and-test#use-http-get-requests

Reply:
Thanks for the explanation 👍